### PR TITLE
Project settings page

### PR DESCRIPTION
**Pull request by chrischo360**
# Description

> Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Checklist

- [ ] This PR can be reviewed in under 30 minutes
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have assigned reviewers to this PR.

### CODE REVIEW

**Review by cbolles — DISMISSED** [review]

File: packages/client-admin/src/App.tsx [hunk 1]
```diff
@@ -24,6 +25,7 @@ export const App: FC = () => {
                   <Route element={<AdminGuard />}>
                     <Route path={Paths.HOME} element={<Home />} />
                     <Route path={Paths.USER_LIST} element={<Users />} />
+                    <Route path={Paths.PROJECT_SETTINGS} element={<ProjectSettings />} />
```
Comment (gregfrasco) [inline packages/client-admin/src/App.tsx]: Could this just be `/project`

File: packages/client-admin/src/pages/projectsettings.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+  return (
+    <MuiIcon fontSize={size}>
+      <img src={icon} alt="icon" style={{ width: imageSize, height: imageSize }} />
+    </MuiIcon>
+  );
+};
+
+export const ProjectSettings = () => {
+  const { token, decoded_token, setToken } = useAuth();
```
Comment (gregfrasco) [inline packages/client-admin/src/pages/projectsettings.tsx]: Just need decoded Token
```suggestion
  const { decoded_token } = useAuth();
```

File: packages/client-admin/src/pages/projectsettings.tsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  const projectId = decoded_token?.projectId || '';
+  const [isDisabled, setIsDisabled] = useState(true);
```
Comment (gregfrasco) [inline packages/client-admin/src/pages/projectsettings.tsx]: I would change it to isEditing instead, to be more clear about what this variable does

File: packages/client-admin/src/pages/projectsettings.tsx [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    >
+      {({ isSubmitting, values }) => (
+        <Form>
+          <Card>
+            <CardHeader
+              action={
+                <IconButton disabled={isSubmitting} onClick={() => setIsDisabled(false)}>
+                  Edit
+                  <Icon icon={faCaretDown} />
+                </IconButton>
```
Comment (gregfrasco) [inline packages/client-admin/src/pages/projectsettings.tsx]: Can't really have text in the Icon Button.
You can really un-edit, on click should more be a toggle

```suggestion
                <IconButton onClick={() => toggleEdit()}>
                  <Icon icon={isEditing ? faCircleXmark : faPen} />
                </IconButton>
```

File: packages/client-admin/src/pages/projectsettings.tsx [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+              }
+            />
+            <CardContent>
+              <FormLabel>Name</FormLabel>
```
Comment (gregfrasco) [inline packages/client-admin/src/pages/projectsettings.tsx]: Pull in this component to uses as text input
https://github.com/hicsail/authentication-service/blob/main/packages/client/src/components/forms/text-input.tsx

File: packages/client-admin/src/pages/projectsettings.tsx [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+                    <IconPreview icon={values.logo} size="large" imageSize={100} />
+                  </Box>
+                )}
+              </Field>
+              <FormLabel>Home Page</FormLabel>
+              <Field name="homePage">{({ field }: { field: any }) => <TextField fullWidth {...field} disabled={isDisabled} />}</Field>
+              <FormLabel>Redirect URL</FormLabel>
+              <Field name="redirectUrl">{({ field }: { field: any }) => <TextField fullWidth {...field} disabled={isDisabled} />}</Field>
+            </CardContent>
+            <IconButton type="submit" disabled={isSubmitting || isDisabled}>
```
Comment (gregfrasco) [inline packages/client-admin/src/pages/projectsettings.tsx]: The save button can be a regular button, look at the submit button in client

**Review by gregfrasco — CHANGES_REQUESTED** [review]

**Review by gregfrasco — APPROVED** [review]

**Pull request merged**